tests/common/plugins/conditional_mark/tests_mark_conditions_platform_tests.yaml

```diff
@@ -71,6 +71,34 @@ platform_tests/api/test_chassis.py::TestChassisApi::test_get_watchdog:
     conditions:
       - "'sw_to3200k' in hwsku"
 
+platform_tests/api/test_chassis.py::TestChassisApi::test_get_model:
+  skip:
```

Comment:
@assrinivasan should change skip to xfail?

Reply:
@assrinivasan Another thing is the cases in conditions yaml file should be alphabetical order.